package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "fitter",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "client": "webpack --watch",
```

Comment:
@Sim0nSl1003 This is perfectly fine, but I just want to share (for folks who may not have tried this before) that the watch flag can be omitted here and then defined in the Webpack config file itself! Basically just set "watch" to be "true."